src/components/CurrentCityWeather/CurrentCityWeather.js

```diff
@@ -3,34 +3,41 @@ import "./CurrentCityWeather.css";
 import Autocomplete from "react-google-autocomplete";
 import Background from "../../data/BackGroundAccordingToWeather";
 import { Col, Row } from "react-bootstrap";
+import Map from "../../components/Map/Map";
 
 const CurrentCityWeather = () => {
   const [error, setError] = useState(null);
   const [isLoaded, setIsLoaded] = useState(false);
   const [city, setCity] = useState("New York City");
   const [results, setResults] = useState(null);
+  const [lat, setLat] = useState("");
+  const [long, setLong] = useState("");
+  const [cityCoordinates, setCityCoordinates] = useState({lat: '40.7128', lon: '-74.0060'});
 
-  
   useEffect(() => {
     fetch(
-      "https://api.openweathermap.org/data/2.5/weather?q=" +
-        city +
-        "&units=metric" +
-        "&appid=" +
-        process.env.REACT_APP_APIKEY
+      "https://api.openweathermap.org/data/2.5/weather?lat=" +
+          cityCoordinates.lat +
+          "&lon=" +
+          cityCoordinates.lon +
+          "&units=metric" +
+          "&appid=" +
+          process.env.REACT_APP_APIKEY
```

Comment:
While this code works fine, I saw this done recently and have been using it in my own code to make things a little easier:

```suggestion
      "https://api.openweathermap.org/data/2.5/weather?"
      + `lat=${cityCoordinates.lat}&`
      + `lon=${cityCoordinates.lon}&`
      + "units=metric&"
      + `appid=${process.env.REACT_APP_APIKEY}`
```